### DISCUSSION TITLE
CollaborationCursor: disable highlighting

### DISCUSSION THREAD

**Discussion by wagnerpaz** [ueberdosis/tiptap#1851] [General Questions]
Hey guys!

I need to modify the collaboration cursors to accomplish some business requirements.
What I need is to disable the highlighting for users without permission.
I managed to disable cursor caret and cursor label, by overwriting the render function of CollaborationCursor plugin, but looking in the source I can't find where it handles highlighting.

Can you help me? thanks!

**Comment by ryanb** [comment]
@wagnerpaz there's a new `selectionBuilder` option on `y-prosemirror`, but it's not released yet and there's currently no way to set it through CollaborationCursor extension. I've added a request for this here: https://github.com/ueberdosis/tiptap/issues/2593
